src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/Public.Data/PreviewTokenController.cs

```diff
@@ -36,4 +36,14 @@ public class PreviewTokenController(IPreviewTokenService previewTokenService) :
             .GetPreviewToken(previewTokenId, cancellationToken)
             .HandleFailuresOrOk();
     }
+
+    [HttpDelete("{previewTokenId:guid}")]
```

Comment:
My only comment here is that I would've personally expected a POST / PUT / PATCH rather than a DELETE, as I'd imagine that I'd be surprised to call a DELETE API endpoint but still see the entity I'd deleted in the `List` endpoint.

Reply:
Good point, I originally considered `PATCH` as the most appropriate as the `Expiry` field is updated, but since the caller won't be providing the value of `Expiry` and there will be no body, I ruled it out and went with `DELETE`.

I've updated this now from `DELETE` to `POST`, changing the request from 

```
DELETE https://localhost:5021/api/public-data/preview-tokens/00000000-0000-0000-0000-000000000000
```

to

```
POST https://localhost:5021/api/public-data/preview-tokens/00000000-0000-0000-0000-000000000000/revoke
```

Do you think the response should contain the updated resource or be empty (gone with empty for now)?

Reply:
I think I might've expected the updates resource in the response, but tbh empty is fine for me as well.

Reply:
Response added in! 🙂